tests/devapps/LinuxApp/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.Identity.Client;
+
+using System;
+
+ 
+
+namespace CrossPlatformConsoleApp
+
+{
+
+    class Program
+
+    {
+
+        public static string ClientID = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842"; //msidentity-samples-testing tenant
+
+        public static string[] Scopes = { "User.Read" };
+
+        static void Main(string[] args)
+
+        {
+
+            Console.WriteLine("Hello World!");
+
+ 
+
+            var pcaBuilder = PublicClientApplicationBuilder.Create(ClientID)
+
+                .WithRedirectUri("http://localhost")
+
+                .Build();
+
+ 
+
+            AcquireTokenInteractiveParameterBuilder atparamBuilder = pcaBuilder.AcquireTokenInteractive(Scopes);
+
+ 
+
+            AuthenticationResult authenticationResult = atparamBuilder.ExecuteAsync().GetAwaiter().GetResult();

```

Comment:
nit: console apps can also use `static async Task Main` async entrypoint, to avoid GetAwaiter stuff.

Reply:
Unfortunately they do not work: (
```CSC : error CS5001: Program does not contain a static 'Main' method suitable for an entry point```

Reply:
Here's an example of this working: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/tests/devapps/NetFxConsoleTestApp/Program.cs#L91

Reply:
In any case, maybe it's easier to reuse `NetCoreTestApp` for testing this ? It's a netcore app and it already calls AcquireTokenInteractive.

Reply:
Removed the App and one can use the package in the sample app updated in the notes.